lib/Elevate/Blockers/Leapp.pm

```diff
@@ -34,6 +34,7 @@ sub check ($self) {
     my $blockers = $self->cpev->leapp->search_report_file_for_blockers(
         qw(
           check_installed_devel_kernels
+          cl_mysql_repository_setup
```

Comment:
What does this do?

Reply:
It is the name of the `actor` in `leapp-report.json` that has an inhibitor.  This tells the `leapp preupgrade` check that it is safe to ignore this inhibitor (do not block on it) since we address the issue via a component.